cluster/node-pools/worker-combined/stack.yaml

```diff
@@ -132,6 +132,14 @@ Resources:
             Value: kubernetes
           - Key: component
             Value: "shared-resource"
+        - ResourceType: "network-interface"
+          Tags:
+          - Key: application
+            Value: kubernetes
+          - Key: component
+            Value: "shared-resource"
```

Comment:
Is this a known `component` value? Or are we introducing this only for interfaces?

Reply:
Ah, I see we used a similar value for volumes [here](https://github.com/zalando-incubator/kubernetes-on-aws/pull/6712/files#diff-e46dbad970101fad1d15cfac46cf072cd7ed6fb5ebb4d160a485a58cb62700cfR133).

Reply:
Yeah, we use it for all "worker node resources" just to indicate this is not totally owned by the `kubernetes` component especially in terms of cost.

Reply:
@mikkeloscar do we use the same for ALB/NLB/ELB?

Reply:
@szuecs for ALB/NLB from ingress the `application` tag is `kube-ingress-aws-controller` and there is no component defined. The ENIs don't seem to be tagged at all when created for a load balancer :/

For service Type=LoadBalancer users can set the application and component labels on the service resource and we add those as tags on the LB.